api/internal/handlers/kubernetes/install.go

```diff
@@ -128,15 +129,65 @@ func (h *Handler) GetInfraStatus(w http.ResponseWriter, r *http.Request) {
 //	@Tags			kubernetes-install
 //	@Security		bearerauth
 //	@Produce		json
-//	@Success		200	{object}	v1beta1.Config
+//	@Success		200	{object}	kotsv1beta1.Config
 //	@Router			/kubernetes/install/app/config [get]
 func (h *Handler) GetAppConfig(w http.ResponseWriter, r *http.Request) {
-	config, err := h.installController.GetAppConfig(r.Context())
+	// import is used in swagger annotation above
+	_ = kotsv1beta1.Config{}
```

Comment:
Should this be outside of the handler? It's probably cheap enough but we're creating a config struct every single time on request.